README.md

```diff
@@ -0,0 +1,75 @@
+<!-- ABOUT THE PROJECT -->
+
+# Skydive Operator (WIP)
+
+This is a simple operator to deploy skydive analyzer and agents.
+
+<!-- GETTING STARTED -->
+
+## Getting Started
+
+To set up this operator follow the instructions below:
+
+### Prerequisites
+
+* Make sure you have golang installed on your machine, The go-lang version that this operator was built on is go1.15.7
+  darwin/amd64
+
+* An openshift cluster
+    * If you wish to run this locally by using kind please
+      check [KindInstallationREADME.md](hack/KindInstallationREADME.md)
+
+### Modifying the CRD
```

Comment:
move after installation section (and you can call this section Customize skydive deployment / CRD)